ui/src/ModelsList/index.js

```diff
@@ -4,14 +4,9 @@ import { ListGroupItem, ListGroup, Badge, Container } from "reactstrap";
 import { Link } from "react-router-dom";
 
 export default class ModelsList extends Component {
-  constructor(props) {
-    super(props);
+  state = { models: [], target: [] };
```

Comment:
this is equivalent to `constructor(props)` ?

Reply:
not really equivalent, you'd still use a constructor if you need to do something when creating a class, but in this case we just need a property on the class.